.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```

Comment:
Be careful with the files that you add to Git - this one is arguably useful (potentially there are other developers working on a project who maybe want to be using a consistent port), however watch out for stuff like `.DS_Store` - this is a blob of Operating System data that's only really useful for your machine, so we want to stop this sort of stuff leaking into working repos.

This is where `.gitignore` is our friend, and there's a good blog article about it here if you haven't used it before: https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/